src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -62,6 +66,31 @@ export default class DataTable extends Vue {
   _paginationListenersAdded = false;
   _chiMajorVersion = 5;
 
+  _toggleInfoPopover(infoPopoverRef: string) {
+    const popover = this.$refs[infoPopoverRef];
+
+    if (popover) {
+      // eslint-disable-next-line
+      // @ts-ignore
+      popover.toggle();
+    }
+  }
+
+  _getDescription(description: string | DataTableColumnDescription) {
+    const template = (description as DataTableColumnDescription).template;
```

Comment:
So, you accept description as `string` or `DataTableColumnDescription`, but here you read it always as `DataTableColumnDescription`?
Why not check
```
if (typeof description === 'string') {
...
} else {
    const template = (description as DataTableColumnDescription).template;
...
}
```
?

Reply:
oh it's just an alternative way to check the type that i use (cause if (template) is false it means it's a string), but the way you suggest is probably closer to a best practice for this so will change it, thanks!

Reply:
actually no, i see what you mean, fixed!